.github/workflows/internal-images.yml

```diff
@@ -24,9 +26,13 @@ jobs:
       matrix:
         # Be precise in tag versions to improve reproducibility
         include:
-          - dockerfile: |  # Don't install golang as the version often changes
+          # This installs the dependencies needed to build func-e including the latest patch of go
+          - dockerfile: |
               FROM centos:8
-              RUN yum install -y --quiet make which git gcc && yum clean all
+              RUN yum install -y --quiet make which git gcc && yum clean all && \
+                  curl -sL https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | GIMME_GO_VERSION=${{ env.GO_VERSION }}.x bash && \
+                  source ~/.gimme/envs/latest.env && \
```

Comment:
I think this only applies to the current RUN directive and won't effect the path of the next, so maybe this isn't ready for merge, yet!